src/BenchmarkDotNet/Toolchains/InProcess.Emit.Implementation/Runnable/RunnableReuse.cs

```diff
@@ -102,7 +102,8 @@ private static IEngineFactory GetEngineFactory(BenchmarkCase benchmarkCase)
                 TargetJob = benchmarkCase.Job,
                 OperationsPerInvoke = benchmarkCase.Descriptor.OperationsPerInvoke,
                 MeasureGcStats = benchmarkCase.Config.HasMemoryDiagnoser(),
-                Encoding = benchmarkCase.Config.Encoding
+                Encoding = benchmarkCase.Config.Encoding,
+                BenchmarkName = FullNameProvider.GetBenchmarkName(benchmarkCase)
```

Comment:
Same as in https://github.com/dotnet/BenchmarkDotNet/blob/f10752ace1bf4c5e4316c6f9b7b34ca25ac46c19/src/BenchmarkDotNet/Toolchains/InProcess/InProcessRunner.cs#L151